utils/node_local_test.py

```diff
@@ -1359,6 +1368,10 @@ def test_pydaos_kv(server, conf):
     kv = None
     print('Closing container and opening new one')
     kv = container.get_kv_by_name('my_test_kv')
+    kv = None
+    container = None
+    daos._cleanup()
```

Comment:
(pylint-protected-access) Access to a protected member _cleanup of a client class